source/github-helpers/index.ts

```diff
@@ -37,6 +37,34 @@ export const getCurrentCommittish = (pathname = location.pathname, title = docum
 	return unslashedCommittish;
 };
 
+export const getCurrentBranchFromFeedLink = (): string | undefined => {
+	// .last needed for #2799
+	const feedLink = select.last('link[type="application/atom+xml"]');
+	if (!feedLink) {
+		return;
+	}
+
+	return new URL(feedLink.href)
+		.pathname
+		.split('/')
+		.slice(4) // Drops the initial /user/repo/route/ part
+		.join('/')
+		.replace(/\.atom$/, '');
+};
+
+export const getCurrentBranchFromFindLink = async (): Promise<string | undefined> => {
+	if (pageDetect.isPRFiles()) {
```

Comment:
```suggestion
	// On `isPRFiles` the `t` shortcut refers to the search file dropdown  
	if (pageDetect.isPRFiles()) {
```

Reply:
s/refers to/opens/

Reply:
Actually, this link only natively exists on some pages, so we should just target them directly. I think they're isTree and is isSingleFile?

Reply:
All pages that I have tested

Reply:
Actually… isn't this ever only needed on the root? Everything else on those pages is already covered by `document.title` in `getCurrentCommittish`

Reply:
I think you maybe right!